polaris-react/src/components/SkeletonTabs/SkeletonTabs.scss

```diff
@@ -18,7 +18,7 @@
     bottom: -1px;
     left: var(--p-space-3);
     right: var(--p-space-3);
-    height: var(--p-border-width-3);
+    height: 3px;
```

Comment:
I understand this is a height. However it is rendering a line under the tab. I don't mind this using the border token because we would actually want this to change if the border got updated.